lib/pg_query/deparse.rb

```diff
@@ -332,6 +336,20 @@ def deparse_bool_expr_not(node)
       format('NOT %s', deparse_item(node['args'][0]))
     end
 
+    def deparse_boolean_test(node)
+      test = case node['booltesttype']
+             when BOOLEAN_TEST_TRUE
+               ' IS true'
+             when BOOLEAN_TEST_NOT_TRUE
+               ' IS NOT true'
+             when BOOLEAN_TEST_FALSE
+               ' IS false'
+             when BOOLEAN_TEST_NOT_FALSE
+               ' IS NOT false'
```

Comment:
Given that https://www.postgresql.org/docs/9.5/static/functions-comparison.html has true/false uppercase (i.e. TRUE/FALSE), I'd be more inclined to deparse to uppercase as well.